price-server/src/server/mod.rs

```diff
@@ -135,9 +135,23 @@ impl Price for PriceService {
 
     async fn get_cents_per_sats_exchange_mid_rate(
         &self,
-        _request: Request<GetCentsPerSatsExchangeMidRateRequest>,
+        request: Request<GetCentsPerSatsExchangeMidRateRequest>,
     ) -> Result<Response<GetCentsPerSatsExchangeMidRateResponse>, Status> {
-        Err(Status::unimplemented(""))
+        let req = request.into_inner();
+
+        let mid_amount_in_cents = self
+            .app
+            .get_cents_per_sats_exchange_mid_rate(Sats::from_major(req.amount_in_satoshis))
```

Comment:
I'm confused about this... the proto definition says it doesn't have a field:
```
message GetCentsPerSatsExchangeMidRateRequest {}
message GetCentsPerSatsExchangeMidRateResponse {
  double ratio_in_cents_per_satoshis = 1;
}
```
How does this compile?